### PR TITLE
Add proto validation for integers

### PR DESCRIPTION
**Pull request by rdimitrov**
# Summary

Ref https://github.com/stacklok/minder-stories/issues/94

## Change Type

***Mark the type of change your PR introduces:***

- [x] Bug fix (resolves an issue without affecting existing features)
- [ ] Feature (adds new functionality without breaking changes)
- [ ] Breaking change (may impact existing functionalities or require documentation updates)
- [ ] Documentation (updates or additions to documentation)
- [ ] Refactoring or test improvements (no bug fixes or new functionality)

# Testing

***Outline how the changes were tested, including steps to reproduce and any relevant configurations. 
Attach screenshots if helpful.***

# Review Checklist:

- [x] Reviewed my own code for quality and clarity.
- [ ] Added comments to complex or tricky code sections.
- [ ] Updated any affected documentation.
- [ ] Included tests that validate the fix or feature.
- [ ] Checked that related changes are merged.

### CODE REVIEW

**Review by eleftherias — DISMISSED** [review]

**Review by eleftherias — APPROVED** [review]

File: proto/minder/v1/minder.proto [hunk 1]
```diff
@@ -93,7 +93,10 @@ message Cursor {
     // an per-rpc basis.
     string cursor = 1;
     // size is the number of items to retrieve from the collection.
-    uint32 size = 2;
+    uint32 size = 2 [
+        (buf.validate.field).ignore_empty = true,
```
Comment (evankanderson) [inline proto/minder/v1/minder.proto:97]: `ignore_empty` seems to be deprecated in favor of:

```
(buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
```

For optional fields, I think you want:

```
(buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
```

Which allows the zero-value as well as "not set at all".

Comment (evankanderson) [reply proto/minder/v1/minder.proto:97]: Ref: https://buf.build/bufbuild/protovalidate/docs/main:buf.validate#buf.validate.FieldConstraints

Comment (rdimitrov) [reply proto/minder/v1/minder.proto:97]: oh, I'll fix it right away, copilot auto-completion strikes again

Comment (rdimitrov) [reply proto/minder/v1/minder.proto:97]: Ref. https://github.com/mindersec/minder/pull/4882

**Pull request merged**